.github/ISSUE_TEMPLATE/remarx_software_release.md

```diff
@@ -0,0 +1,39 @@
+---
+name: remarx release checklist
+about: Checklist for releasing new versions of remarx
+title: remarx release checklist
+labels: chore
+assignees: ''
+---
+
+## release prep
+
+- [ ] Pull updated copies of the develop and main branches
+- [ ] Use git flow to create a new release branch with the appropriate version (e.g., `git flow release start 0.5`)
+
+### prep release candidate for acceptance testing
+
+- [ ] Update the version number to the appropriate release candidate number (e.g., `0.5.rc1`)
+- [ ] Create a draft PR (since it should not be merged)
+- [ ] Review the changelog to make sure that all features, changes, bugfixes, etc included in the release are documented. You may want to review the git revision history to be sure you've captured everything.
+- [ ] Review the README to make sure that its contents are up to date
+- [ ] Check python requirements for any internal dependencies that should be released (or at least pinned to a specific git commit)
+- [ ] Confirm that all checks for the draft PR pass (e.g., unit tests, code coverage checks)
+- [ ] Review code documentation to make sure it is up to date.
+- [ ] Use git flow to finish the release (merge release branch into both main and develop, create a tag, remove the release branch, etc.). (`git flow release finish 0.5`)
```

Comment:
I think this line should either be removed or should replace the last step in the section under "acceptance testing passes"